doc/release/platforms/README.cray

```diff
@@ -420,52 +489,120 @@ To use this combination of runtime layers:
    of the total locale memory being wasted.
 
 
-Special parameters for the Cray-specific runtime layers
--------------------------------------------------------
+Due to the use of hugepages in the ugni comm layer, tasking layers
+cannot use guard pages for stack overflow detection.  Qthreads tasking
+uses uses guard pages for its stack overflow detection, so when ugni
+communications and qthreads tasking are combined, task call stack
+overflow detection is turned off.  This occurs automatically and nothing
+needs to be done to enable it.  (Muxed tasking uses another technique to
```

Comment:
the word "enable" is a little strange here since it's talking about enabling turning something off.